### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by Kaweesi-Matia**
## Below is the description of the PR

- [ ] Use methods that you have created in the [Blog App - processing data in models]
- [ ] Implement the design from the sneak peek wireframes
- [ ] Modify ReadMe file

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1,2 +1,11 @@
 <h1> Home Page </h1>
-<p> List of all users </p>
\ No newline at end of file
+<p> List of all users </p>
```
Comment (codecaiine) [inline app/views/users/index.html.erb]: - [ ] Good job so far but please make sure that the index view page design match with the given [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png) as required for this project :+1:

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,9 @@
 <h1>User details page </h2>
 <p>Details of the user which includes both Bio and Posts </h2>
+<h2><%= @user.Name%></h2>
```
Comment (codecaiine) [inline app/views/users/show.html.erb]: - [ ] As I said above :up: Please make sure that by clicking on each user , each user show page is looks like  the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png) as required for this project  :+1:

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1,2 +1,9 @@
...
+<p><%= @user.Bio%></p>
+<p><%= @user.PostsCounter%></p>
+<%=link_to "Back", users_path%>
+<%=link_to "Posts", user_posts_path(@user)%>
```
Comment (codecaiine) [inline app/views/users/show.html.erb]: - [ ] Please make sure that your **See all Post** links takes to user's post index and is displaying like on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png) as required for this project :+1:

File: app/views/posts/index.html.erb [hunk 4]
```diff
@@ -1,2 +1,12 @@
 <h1>Posts page </h1>
-<p> List of users posts here </p>
\ No newline at end of file
+<p> List of users posts here </p>
+<% @posts.each do |post|  %>
```
Comment (codecaiine) [inline app/views/posts/index.html.erb]: - [ ] According to the project requirement you should  uses `post.recent_comments` to render 5 most recent comments . Please fix it in order to make your app work as expected :+1:

File: app/views/posts/show.html.erb [hunk 5]
```diff
@@ -1,2 +1,8 @@
-<h1> User posts page </h1>
-<p> Details of a specific post of a user</p>
\ No newline at end of file
+<h1> User post page </h1>
```
Comment (codecaiine) [inline app/views/posts/show.html.erb]: - [ ] Please make sure that your post show page displays post with all comments listed like  on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png) as required for this project :+1:

File: app/views/users/show.html.erb (hunk 2 above)
Comment (codecaiine) [inline app/views/users/show.html.erb]: - [ ] According to the project requirement you should use `#recent_post` method to render the user's 3 most recent post. Please make sure to fix it and make your app work as expeced :+1:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Kaweesi-Matia , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Great view setup :heavy_check_mark: 
- No rubocop errors :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/index.html.erb (hunk 1 above)
Comment (Kaweesi-Matia) [reply app/views/users/index.html.erb]: Make the index view page match the figma

File: app/views/users/show.html.erb (hunk 2 above)
Comment (Kaweesi-Matia) [reply app/views/users/show.html.erb]: resolve the issue

File: app/views/users/show.html.erb (hunk 3 above)
Comment (Kaweesi-Matia) [reply app/views/users/show.html.erb]: Make See all Post linking to the user index

**Review by AnselemOdims — APPROVED** [review]
### Hi  @Kaweesi-Matia 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**